src/utils/package.ts

```diff
@@ -57,10 +57,6 @@ export function formatDateDistance(lastUpdate): string {
   return distanceInWordsToNow(new Date(lastUpdate));
 }
 
-export function buildScopePackage(scope: string, packageName: string): string {
-  return `@${scope}/${packageName}`;
```

Comment:
We need this method.

Reply:
Hmmm it's not being used anywhere else..only in the Version Page